### PR TITLE
Register cibucristi.is-a.dev

### PR DESCRIPTION
**Pull request by cristiancibu**
Register cibucristi.is-a.dev with TXT record pointing to vc-domain-verify=cibucristi.is-a.dev,6d7812c190dbf7b6f851.

### CODE REVIEW

File: domains/cibucristi.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "cibucristi",
+           "email": "cibucristi1@gmail.com",
+           "discord": "773576280150900749"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=cibucristi.is-a.dev,6d7812c190dbf7b6f851"
```
Comment (CuteDog5695) [inline domains/cibucristi.json:9]: we dont support Vercel at the moment

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by cibucristi** [comment]
I am using vercel
image

**Comment by cibucristi** [comment]
what about netlify?

**Comment by CuteDog5695** [comment]
> what about netlify?

also not allowed at the moment

**Comment by CuteDog5695** [comment]
> do you guys have any free web host that you support?

Cloudflare Pages and Github Pages are an option

**Comment by cibucristi** [comment]
ok, i'll deploy it to github pages rn

**Comment by CuteDog5695** [comment]
> but if i buy a web host can you activate it there?

you can try to use a webhost for is-a.dev

**Comment by cibucristi** [comment]
I have deploy it to cloudflare pages.

**Comment by cibucristi** [comment]
CNAME record:

**Name**: cibucristi
**Target**: cibucristi-dev.pages.dev

**Comment by CuteDog5695** [comment]
> CNAME record:
> 
> **Name**: cibucristi **Target**: cibucristi-dev.pages.dev

edit the file to have the CNAME record there (reffer to here on how to replace TXT record with CNAME record https://www.is-a.dev/docs/domain_structure/)

**Comment by cibucristi** [comment]
Done

**Review by iostpa — APPROVED** [review]
seems good to me now (also nice website!)

**Pull request merged**